server.js

```diff
@@ -74,6 +74,22 @@ app.get('/customConfig', (req, res) => {
   });
 });
 
+// Dynamic static path for walkthrough assets. Based on the walkthrough ID
+// provided it'll look in different paths.
+app.get('/walkthroughs/:walkthroughId/files/*', (req, res) => {
+  const {
+    params: { walkthroughId }
+  } = req;
+  const file = req.param(0);
+  const walkthrough = walkthroughs.find(wt => wt.id === walkthroughId);
+  if (!walkthrough) {
+    return res.status(404).json({ error: `Walkthrough with ID ${walkthroughId} is not found` });
+  }
+  // Dotpaths are not allowed by default, meaning an end-user shouldn't be able
+  // to abuse the file system using the wildcard file param.
+  res.sendFile(`${walkthrough.basePath}/${file}`);
```

Comment:
See below comment for why it's best to change this to something like here: https://github.com/integr8ly/tutorial-web-app/pull/315/files#diff-78c12f5adc1848d13b1c6f07055d996eR101